devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -617,14 +619,11 @@ void prepare() throws Exception {
 
         public void run() throws Exception {
             // Display the launch command line in dev mode
-            getLog().info("Launching JVM with command line: " + args.toString());
-            ProcessBuilder pb = new ProcessBuilder(args.toArray(new String[0]));
-            pb.redirectError(ProcessBuilder.Redirect.INHERIT);
-            pb.redirectOutput(ProcessBuilder.Redirect.INHERIT);
-            pb.redirectInput(ProcessBuilder.Redirect.INHERIT);
-            pb.directory(workingDir);
-            process = pb.start();
-
+            getLog().info("Launching JVM with command line: " + args.stream().collect(joining(" ")));
```

Comment:
Actually, I think this should be using the debug level instead. It doesn't seem to be a really useful information to be displayed everytime

Reply:
Hello @gastaldi, this was intentionally changed to `INFO` as part of https://github.com/quarkusio/quarkus/pull/5366 to help quickly notice any obvious issues for cases noted in that PR.

Reply:
Okay, I reverted my commit that undo'ed that. Thanks for the pointer :)